win32/deluge-bbfreeze.py

```diff
@@ -84,7 +83,6 @@ def ignored_files(adir, filenames):
 # copy gtk theme files
 theme_include_list = [
     [gtk_root, "share/icons/hicolor/index.theme"],
-    [gtk_root, "lib/gtk-2.0/2.10.0/engines"],
```

Comment:
This surely means the entire `DelugeStart Theme` needs converting/removing/replacing?